ios/engine/KMEI/KeymanEngine/Classes/Keyboard/KeymanWebViewController.swift

```diff
@@ -749,16 +749,6 @@ extension KeymanWebViewController: KeymanWebDelegate {
     }
   }
 
-  func updateShowBannerSetting() {
-    let userData = Storage.active.userDefaults
-    let alwaysShow = userData.bool(forKey: Key.optShouldShowBanner)
```

Comment:
Now that this setting is no longer supported, it seems the key be removed from userDefaults when we see it. Is that cleaned up anywhere or is it an ignored remnant that lives on?

Reply:
Currently:  ignored remnant.  It'd only matter if we decide to use it again in some form at some point down the road.

Reply:
I don't know if there is a good place to document this, but it would be a good comment to include somewhere in the code. The userdefaults is kind of like our settings document but with an undocumented file format, so it would be good to know what things could be floating around in there if anyone ever is trying to decipher real user data.

Reply:
Yeah... that was the intent behind leaving the deprecated key noted here: https://github.com/keymanapp/keyman/pull/10929#discussion_r1542337274 - that _is_ in the "Settings-related keys" section, after all.